src/main/java/org/quantumbadger/redreader/reddit/url/UserPostListingURL.java

```diff
@@ -22,40 +22,43 @@
 import org.quantumbadger.redreader.R;
 import org.quantumbadger.redreader.common.Constants;
 import org.quantumbadger.redreader.common.General;
+import org.quantumbadger.redreader.reddit.PostSort;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class UserPostListingURL extends PostListingURL {
 
 	public static UserPostListingURL getSaved(String username) {
-		return new UserPostListingURL(Type.SAVED, username, null, null, null);
+		return new UserPostListingURL(Type.SAVED, username, null, null, null, null);
 	}
 
 	public static UserPostListingURL getHidden(String username) {
-		return new UserPostListingURL(Type.HIDDEN, username, null, null, null);
+		return new UserPostListingURL(Type.HIDDEN, username, null, null, null, null);
 	}
 
 	public static UserPostListingURL getLiked(String username) {
-		return new UserPostListingURL(Type.UPVOTED, username, null, null, null);
+		return new UserPostListingURL(Type.UPVOTED, username, null, null, null, null);
 	}
 
 	public static UserPostListingURL getDisliked(String username) {
-		return new UserPostListingURL(Type.DOWNVOTED, username, null, null, null);
+		return new UserPostListingURL(Type.DOWNVOTED, username, null, null, null, null);
 	}
 
 	public static UserPostListingURL getSubmitted(String username) {
-		return new UserPostListingURL(Type.SUBMITTED, username, null, null, null);
+		return new UserPostListingURL(Type.SUBMITTED, username, null, null, null, null);
 	}
 
 	public final Type type;
 	public final String user;
+	public final PostSort order;
 	public final Integer limit;
 	public final String before, after;
 
-	UserPostListingURL(Type type, String user, Integer limit, String before, String after) {
+	UserPostListingURL(Type type, String user, PostSort order, Integer limit, String before, String after) {
 		this.type = type;
 		this.user = user;
+		this.order = order;
```

Comment:
Might be an idea to check for `RISING` in this constructor, and default to something else.